test/e2e/framework/plugin_lifecycle_operations.go

```diff
@@ -62,11 +62,20 @@ type PluginGroupOps interface {
 	InstallPluginsFromGroup(pluginNameORAll, groupName string, opts ...E2EOption) error
 }
 
+type PluginDownloadAndUploadOps interface {
+	// DownloadPluginBundle downloads the plugin inventory and plugin bundles to local tar file
+	DownloadPluginBundle(image string, groups []string, toTar string, opts ...E2EOption) error
+
+	// UploadPluginBundle performs the uploading plugin bundle to the remote repository
+	UploadPluginBundle(toRepo string, tar string, opts ...E2EOption) error
```

Comment:
Could you kindly update the document and clarify whether it should override or append the existing repository?